src/Components/Patient/PatientHome.tsx

```diff
@@ -1110,6 +1121,71 @@ export const PatientHome = (props: any) => {
               </div>
             </div>
           </div>
+          <div className="w-full">
+            <div className="h-full space-y-2 rounded-lg bg-white p-7 shadow">
+              <div className="border-b border-dashed pb-2 text-xl font-bold text-gray-900">
+                Insurance Details
+              </div>
+
+              {insuranceDetials?.count == 0 || insuranceDetailsLoading ? (
+                <div className="flex w-full items-center justify-center text-xl font-bold text-gray-500">
+                  No Insurance Data Available
+                </div>
+              ) : (
+                <div className="grid grid-cols-1 gap-2 sm:grid-cols-2 md:grid-cols-2">
+                  <div className=" ">
+                    <div className="text-sm font-semibold leading-5 text-zinc-400">
+                      Member ID
+                    </div>
+                    <div className="mt-1 whitespace-normal break-words text-sm font-medium leading-5">
+                      {insuranceDetials?.results[0]?.subscriber_id || ""}
+                    </div>
+                  </div>
+                  <div className=" ">
+                    <div className="text-sm font-semibold leading-5 text-zinc-400">
+                      Policy ID / Policy Name
+                    </div>
+                    <div className="mt-1 whitespace-normal break-words text-sm font-medium leading-5">
+                      {insuranceDetials?.results[0]?.policy_id || ""}
+                    </div>
+                  </div>
+                  <div className="sm:col-span-1">
+                    <div className="text-sm font-semibold leading-5 text-zinc-400">
+                      Insurer ID
+                    </div>
+                    <div className="mt-1 whitespace-normal break-words text-sm font-medium leading-5">
+                      {insuranceDetials?.results[0]?.insurer_id || ""}
+                    </div>
+                  </div>
+                  <div className="sm:col-span-1">
+                    <div className="text-sm font-semibold leading-5 text-zinc-400">
+                      Insurer Name
+                    </div>
+                    <div className="mt-1 whitespace-normal break-words text-sm font-medium leading-5">
+                      {insuranceDetials?.results[0]?.insurer_name || ""}
+                    </div>
+                  </div>
```

Comment:
Refactor this into a reusable component since it is identical to the implementation in `InsuranceDetails.tsx`